templates/app-ts/test/plugins/support.test.ts

```diff
@@ -0,0 +1,11 @@
+import { test } from 'tap'
+const Fastify = require('fastify')
```

Comment:
this is the only one left with the require statement
I could not find a better way to make it work honestly :sweat_smile:

Reply:
@Ethan-Arrowood can you chime in?

Reply:
This is still on Fastify v2 -- try using `import * as Fastify from 'fastify'` and see if you can get it to work with that. You may need to define your server with `const fastify = Fastify.fastify()` or something like that though.

FYI all this is fixed in v3 (which is coming very soon)!

Reply:
Actually i found out i needed to import the extended types i created for the custom plugin:
`import Fastify from 'fastify'`
and 
`import types from '../../@types'`